tasks/javascript.js

```diff
@@ -108,6 +108,8 @@ gulp.task('scripts:bundle', () => {
         return browserify(file, { debug: config.isDev })
             .transform(babelify)
             .bundle()
+            .on('error', config.gulp.onError)
```

Comment:
Quick question: gulp plumber is called a few lines down, could we not keep veg that up? Or is s it that the on error listener is required?

Reply:
That’s what I thought too – but unfortunately it doesn’t look like it catches errors within the browserify bundling.  Tried moving it up and it doesn’t work, so this was the only solution I found that stops the watch task from breaking…